web_src/js/features/gitgraph.js

```diff
@@ -46,6 +46,67 @@ export default async function initGitGraph() {
       window.history.replaceState({}, '', window.location.pathname);
     }
   });
+  const url = new URL(window.location);
+  const params = url.searchParams;
+  const updateGraph = () => {
+    const queryString = params.toString();
+    let ajaxUrl = url.toString();
+    if (queryString) {
+      url.search = `?${queryString}`;
+      window.history.replaceState({}, '', `?${queryString}`);
+      ajaxUrl += '&div-only=true';
+    } else {
+      url.search = '';
+      window.history.replaceState({}, '', window.location.pathname);
+      ajaxUrl += '?div-only=true';
```

Comment:
Suggesting to use https://github.com/sindresorhus/query-string instead of piecing together query strings like that.

Reply:
Actually, we can use [browser API](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams) instead, e.g. `new URLSearchParams(window.location.search)`, modify and then `params.toString()`.